src/hooks/c-bridge/useCbridge.ts

```diff
@@ -313,21 +324,72 @@ export function useCbridge() {
     }
   };
 
-  watchEffect(async () => {
-    if (!currentNetworkIdx.value || currentNetworkIdx.value !== null) {
-      await updateBridgeConfig();
+  const handleQuotation = async (): Promise<void> => {
+    if (
+      !quotation.value ||
+      quotation.value === null ||
+      !srcChain.value ||
+      !selectedToken.value ||
+      !amount.value
+    ) {
+      errMsg.value = '';
+      return;
     }
-  });
 
-  watch(
-    [srcChain, destChain, chains],
-    async () => {
-      await watchSelectableChains();
-    },
-    { immediate: true }
-  );
+    const { symbol } = getTokenInfo({
+      srcChainId: srcChain.value.id,
+      selectedToken: selectedToken.value,
+    });
 
-  watchEffect(() => {
+    const balance = Number(await getSelectedTokenBal());
+    const numAmount = Number(amount.value);
+    const { minAmount, maxAmount } = quotation.value;
+    if (!minAmount || !maxAmount) return;
```

Comment:
just curious what possibility of `minAmount` or `maxAmount` has no value?

Reply:
@sirius651 
I haven't met this situation yet. Sure, let me try to remove this line.
Ref: https://gyazo.com/e9343f915329f6e7caca0b0a5fc4a9b6